documentation/source/user_section/tutorials/other-features/spinalcord-smoothing.rst

```diff
@@ -47,4 +47,18 @@ Next, open up your terminal and run the following command:
 After smoothing, the apparent noise is reduced, while the cord edges are preserved, allowing a more reliable segmentation on a second pass.
 
 .. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord-smoothing/io-sct_smooth_spinalcord.png
-   :align: center
\ No newline at end of file
+   :align: center
+
+You can compare the quality of the segmentation produced on the smoothed image by running the following commands:
+
+.. code:: sh
+
+   # Second-pass segmentation using the smoothed anatomical image
+   sct_deepseg_sc -i t1_smooth.nii.gz -c t1 -qc ~/qc_singleSubj
+
+..
+   comment:: Is this really necessary anymore? We have the contrast-agnostic
+             segmentation, which should provide a more accurate segmentation
+             with CSA that is consistent between contrasts. I am wondering
+             how trustworthy the results of ``sct_deepseg_sc`` are on a smoothed
+             spinal cord now that we have a new gold-standard for segmentation.
```

Comment:
I was confused by the fact that we re-segment with `sct_deepseg_sc` after smoothing. I'm wondering if this is even still the state-of-the-art... :thinking:

Reply:
From meeting: smoothing is not necessarily the best method anymore, but it's useful to have an example of it, for the cases where it's important.

In general, should we still use `sct_deepseg_sc`? Probably not, so we should switch to `sct_deepseg spinalcord` (in v7+) everywhere else.